packages/modules/link-modules/src/migration/index.ts

```diff
@@ -125,10 +125,10 @@ export class MigrationsExecutionPlanner implements ILinkMigrationsPlanner {
     `
         )
     )
-      .map(({ table_name }) => table_name)
+      .map(({ table_name }) => table_name.toLowerCase())
       .filter((tableName) =>
         this.#linksEntities.some(
-          ({ entity }) => entity.meta.collection === tableName
+          ({ entity }) => entity.meta.collection.toLowerCase() === tableName
```

Comment:
If the tables were initially created as lower case, I guess we don't need these

Reply:
true